docs/stable/1-client.md

```diff
@@ -94,33 +96,33 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
-
-There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+On top of this we run RLPx-based subprotocols, the client
+MAY use [Whisper v6](https://eips.ethereum.org/EIPS/eip-627), though the client 
+SHOULD use [Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md)
+for privacy-preserving messaging.
 
 #### Node discovery and roles
 
-There are four types of node roles:
-1. Bootstrap nodes
-2. Whisper relayers
-3. Mailservers (servers and clients)
-4. Mobile nodes (Status Clients)
+There are five types of node roles:
+1. `Bootstrap node`
+1. `Whisper/Waku relayer`
```

Comment:
Numbered list format off

Reply:
Also where is the five from?

Reply:
Ah good catch, I had previously split Whisper and Waku relayers into separate node roles. I will change this.

I changed the numbered list so that all are `1.` because markdown parses `1.` as an ordered list. That means that additions to the list don't need to be manually re-sequenced.

Reply:
Huh TIL. So this will render correctly for all (well, ours) kramdown generator?

Reply:
I had to check this, but it appears to already be something some contributors have done, and is working with the live specs.

See https://specs.status.im/spec/2#introduction

> 1 a whisper chat identity keypair
> 2 a set of cryptocurrency wallet keypairs

Which in the raw `.md` file looks like this :

https://github.com/status-im/specs/blob/56d5a2ff3949f3d2a68c70e1ae6780d2a080f16d/docs/stable/2-account.md#L51-L54